src/snowflake/cli/api/exceptions.py

```diff
@@ -162,3 +162,12 @@ def __init__(self, part: str, name: str):
         super().__init__(
             f"{part.capitalize()} provided but name '{name}' is fully qualified name."
         )
+
+
+class IncompatibleParametersError(UsageError):
+    def __init__(self, options: list[str]):
+        options_with_quotes = [f"'{option}'" for option in options]
+        comma_separated_options = ", ".join(options_with_quotes[:-1])
+        super().__init__(
+            f"Parameters {comma_separated_options} and {options_with_quotes[-1]} are incompatible."
```

Comment:
Can you share how this error will look like to users?

Reply:
I attached a screenshot to the description